includes/Core/Assets/Assets.php

```diff
@@ -727,193 +749,4 @@ private function run_before_print_callbacks( WP_Dependencies $dependencies, arra
 			}
 		}
 	}
-
-	/**
-	 * Gets all external assets.
-	 *
-	 * This method should only be called once as it will create a new instance for each asset.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @return array List of Asset instances.
-	 */
-	private function get_external_assets() {
-		$base_url     = $this->context->url( 'dist/assets/' );
-		$script_debug = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
-		$suffix       = $script_debug ? '' : '.min';
-		$react_suffix = ( $script_debug ? '.development' : '.production' ) . $suffix;
-
-		return array(
-			new Script(
-				'lodash',
-				array(
-					'src'          => $base_url . 'vendor/lodash' . $suffix . '.js',
-					'version'      => '4.17.15',
-					'fallback'     => true,
-					'before_print' => function( $handle ) {
-						wp_add_inline_script( $handle, '/*googlesitekit*/ window.lodash = window.lodash || _.noConflict(); window.lodash_load = true;' );
-					},
-				)
-			),
-			new Script(
-				'moment',
-				array(
-					'src'      => $base_url . 'vendor/moment' . $suffix . '.js',
-					'version'  => '2.22.2',
-					'fallback' => true,
-				)
-			),
-			new Script(
-				'react',
-				array(
-					'src'      => $base_url . 'vendor/react' . $react_suffix . '.js',
-					'version'  => '16.11.0',
-					'fallback' => true,
-				)
-			),
-			new Script(
-				'react-dom',
-				array(
-					'src'      => $base_url . 'vendor/react-dom' . $react_suffix . '.js',
-					'version'  => '16.11.0',
-					'fallback' => true,
-				)
-			),
-			new Script(
-				'wp-polyfill',
-				array(
-					'src'          => $base_url . 'js/externals/wp-polyfill.js',
-					'version'      => '7.4.0',
-					'fallback'     => true,
-					'before_print' => function( $handle ) use ( $base_url ) {
-						$inline_polyfill_tests = array(
-							'\'fetch\' in window'                                    => $base_url . 'js/externals/wp-polyfill-fetch.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
-							'document.contains'                                      => $base_url . 'js/externals/wp-polyfill-node-contains.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
-							'window.FormData && window.FormData.prototype.keys'      => $base_url . 'js/externals/wp-polyfill-formdata.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
-							'Element.prototype.matches && Element.prototype.closest' => $base_url . 'js/externals/wp-polyfill-element-closest.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
-							'typeof URL === \'function\''                            => $base_url . 'js/externals/wp-polyfill-url.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
```

Comment:
Are we sure that all of the polyfills we are no longer bundling aren't needed? Perhaps we need to re-add some to our `package.json`? Not just these checks specifically, but any of the polyfills from externals.

Reply:
These are browser polyfills, right? I'd think Babel and webpack together are handling that… and if they aren't we should have them do it rather than separate scripts.

But some of these polyfills (eg `window.fetch`) are for aggressively out-of-date browsers 😄

These are the browsers in our `browserlist`, fwiw:
<img width="676" alt="Screenshot 2020-01-28 at 16 21 54" src="https://user-images.githubusercontent.com/90871/73282974-4ee4ed80-41ea-11ea-99d3-f539a1f053e2.png">